packages/conduit-rxjs-react/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.7.0
+
+Published TBD
+
+1. Updated dependencies to latest version.
+
+2. Updated React lifecycle method name for use with React 16.3 to 18.x. This is a temporary fix as opposed to a more permanent fix discussed in [Issue #3](https://github.com/indiana-university/conduit/issues/3).
```

Comment:
Are we sure that it is "to 18.x" and not "to 17.x"? I couldn't find a source one way or another. But it seems odd that the UNSAFE methods would carry over to a second major version.